src/tests/EdFi.DataManagementService.Tests.E2E/README.md

```diff
@@ -4,33 +4,36 @@ This is a suite of end to end tests that will cover different scenarios. They
 are run against a local DMS container that must be rebuilt to stay in sync with
 the codebase.
 
-## Building the DMS container
+## Testing Process
 
-From the `src` directory, and run:
-`docker build -t local/data-management-service . -f Dockerfile`
-This will copy the current DMS solution into the container, build it and start
-it.
+You may either run the E2E tests locally from docker containers as is done in
+the github actions, or you may run them against your locally debugged API
+instance. See the steps below. 
 
-## Running The Tests
+### Testing locally with docker compose
 
-- Run from the Visual Studio Test Explorer.
-- cd into `/src/tests` and run:
-  `dotnet test EdFi.DataManagementService.Tests.E2E`.
+From the `eng` directory:
 
-## Test Environments
+- If you have stale volumes with old data run `./start-local-dms.ps1 -d -v` to
+  stop services and delete volumes
+- Run `./start-local-dms.ps1 -EnvironmentFile ./.env.e2e`
 
-The tests can be executed against an isolated environment or your current
-running environment.
+### Testing locally with API in debug mode
 
-By default, the tests run against an isolated
-[TestContainers](https://dotnet.testcontainers.org/) environment. This will
-setup the Docker containers for the API and the Backend, see
-[ContainerSetup.cs](./Management/ContainerSetup.cs) for more information.
+To debug the API while running the tests, change `ApiUrl` in
+`OpenSearchContainerSetup.cs` to `http://localhost:5198/` and run
+`EdFi.DataManagementService.Frontend.AspNetCore` in debug mode as usual.
 
-You can also run the tests against your current running environment, by setting
-UseTestContainers to false in the [appsettings](./appsettings.json)
+> [!WARNING] Database Warning 
+> Your database tables will be truncated after each
+> feature file runs. Double check your `DatabaseConnection` in
+> `appSettings.json` and be aware of this before you run the tests.
```

Comment:
@simpat-adam 
May need to mention that Database connection string details should match with values from .env.e2e file (ex: port, username and password).

Reply:
But does it really need to? Wouldn't any valid database connection string be usable - albeit with the warning I mentioned.